src/libraries/Microsoft.PowerFx.Connectors/OpenApiParser.cs

```diff
@@ -58,12 +60,7 @@ internal class OpenApiParser
                     }
 
                     var operationName = op.OperationId ?? path.Replace("/", string.Empty);
-
-                    if (op.RequestBody != null)
-                    {
-                        // RequestBody can be ambiguous- treat as a single record? splat as parameters?
-                        throw new NotImplementedException($"Request body");
-                    }
+                    var bodyType = null as FormulaType;
```

Comment:
>null as FormulaType [](http://example.com/codeflow?start=35&length=19)

?
Is this just: 
FormulaType bodyType = null;

Although doesn't look like we use it